packages/frontend/src/creation-form/components/collaterals/index.tsx

```diff
@@ -96,19 +88,21 @@ export const Collaterals = ({
 
     useEffect(() => {
         if (
-            !pickedToken ||
-            !pickerRawAmount.value ||
-            !pickerRawMinimumPayout.value
+            !state.pickerToken ||
+            !state.pickerAmount ||
+            !state.pickerAmount.value ||
+            !state.pickerMinimumPayout ||
+            !state.pickerMinimumPayout.value
         ) {
             setAddDisabled(true);
             return;
         }
-        const parsedAmount = parseFloat(pickerRawAmount.value);
+        const parsedAmount = parseFloat(state.pickerAmount.value);
         if (data) {
             // check if the user has enough balance of the picked token
```

Comment:
I think this comment can be removed and we can use a variable/function to add this description as a name.

`checkIfEnoughBalance(token)`